sample/kotlin/src/main/kotlin/com/datadog/android/sample/sessionreplay/WebViewRecordFragment.kt

```diff
@@ -29,11 +30,15 @@ internal class WebViewRecordFragment : Fragment() {
         "datadoghq.dev"
     )
 
+    @Volatile
+    private var pageWasLoaded: Boolean = false
```

Comment:
```suggestion
    private var pageIsLoaded: Boolean = false
```

Reply:
I know that the WebView callbacks are being executed on a different thread this is the reason why I want to keep it as @Volatile

Reply:
my suggestion is not about volatile, but about naming